### DISCUSSION TITLE
When will this work on windows?

### DISCUSSION THREAD

**Discussion by WelschCode** [hkchengrex/XMem#84] [General]
When will this work on windows?

**Comment by hkchengrex** [comment]
I have successfully run this on Windows before.

**Comment by WelschCode** [comment]
It's odd when I goto click on the '__init__' for example, it pops up and disappears in the scripts folder. In the Xmem-main folder I click interactive_demo, and any other file and they pop up and disappear. The interactive_demo is the only thing that throws any kind of error, the others are just pure black and vanish. 
FileNotFoundError: [Errno 2] No such file or directory: '.saves/Xmem.pth' which I don't know what to do after this? I don't have a XMem.pth file.

**Reply by hkchengrex** [reply]
Please read our documentation: https://github.com/hkchengrex/XMem/blob/main/docs/DEMO.md
You need to download the model files and you would need to use Python to run the scripts.
